server/command.go

```diff
@@ -10,21 +10,21 @@ import (
 	"github.com/mattermost/mattermost-server/plugin"
 )
 
-const helpText = "###### Mattermost JIRA Plugin - Slash Command Help\n" +
-	"* `/jira connect` - Connect your Mattermost account to your JIRA account\n" +
-	"* `/jira disconnect` - Disonnect your Mattermost account to your JIRA account\n" +
-	"* `/jira instance` - Manage JIRA instances connected to Mattermost\n" +
-	"  * `list` - List known JIRA instances\n" +
-	"  * `select <key or number>` - Select a known instance as current\n" +
-	"  * `add server <URL>` - Add a JIRA Server instance\n" +
-	"  * `add cloud` - Add a JIRA Cloud instance\n" +
+const helpText = "###### Mattermost Jira Plugin - Slash Command Help\n" +
+	"* `/jira connect` - Connect your Mattermost account to your Jira account and subscribe to events\n" +
+	"* `/jira disconnect` - Disonnect your Mattermost account from your Jira account\n" +
+	"* `/jira instance [add/list/select]` - Manage connected Jira instances. Must have System Admin role in Mattermost\n" +
+	"  * `/jira instance add server <URL>` - Connect a Jira Server instance to Mattermost\n" +
```

Comment:
LOL, I added `/jira instance`  for debugging purposes. If we want to keep  these commands we should add permission checks to the code.

Reply:
Cool, I thought it had a permission check. I wouldn't worry about it though, will just remove from the text.

I'm aware it's just for debugging purposes at this point :)